test-project/Cargo.toml

```diff
@@ -8,3 +8,6 @@ authors = ["Thomas Jespersen <laumann.thomas@gmail.com>"]
 [dev-dependencies.compiletest_rs]
 path = ".."
 features = ["tmp"]
+
+[features]
+stable = ["compiletest_rs/stable"]
```

Comment:
Oh, this is how to reference features from dependencies?

Reply:
See the reference here: https://doc.rust-lang.org/cargo/reference/manifest.html#the-features-section

Reply:
Thanks for the pointer!